packages/desktop-gui/cypress/integration/project_nav_spec.js

```diff
@@ -318,8 +318,15 @@ describe('Project Nav', function () {
     })
 
     describe('local storage saved browser', function () {
+      // deliberately not the default 'chrome' browser
```

Comment:
can we clearly have 3 tests here:
- nothing is in localStorage under `choseBrowser`
- old way `chromium`
- new way with stringified object